src/region_map.c

```diff
@@ -1333,6 +1336,7 @@ static const struct MultiPartMapSection sMultiPartMapSections[1] =
     {sEverGrandeCityAreaNames, MAPSEC_EVER_GRANDE_CITY, FLAG_SYS_POKEMON_LEAGUE_FLY},
 };
 
+// what in the hell is this?
```

Comment:
either remove the comment or figure out a proper name for `UnknownStruct3`

Reply:
oops